File: src/main/java/com/my/admin/app/commerce/model/document/PaymentType.java
package com.my.admin.app.commerce.model.document;

import lombok.Data;
import org.springframework.data.annotation.Id;
import org.springframework.data.mongodb.core.mapping.Document;

import java.io.Serializable;

@Document(collection = "payment_types")
@Data
public class PaymentType implements Serializable {

    private static final long serialVersionUID = 1L;

    @Id
    private Long id;

    private String name;

    private String description;
}

File: src/main/kotlin/com/bk/reactive/app/commerce/admin/my/model/document/PaymentType.kt
package com.bk.reactive.app.commerce.admin.my.model.document

import org.springframework.data.annotation.Id
import org.springframework.data.mongodb.core.mapping.Document

@Document(collection = "payment_types")
data class PaymentType(
    @Id
    val id: String,
    val name: String,
    val description: String
)